### ISSUE TITLE
Simon under python 2.7 installs fine but appears blank

### ISSUE THREAD

**Issue by harijay** [hcyrnd/simon#2]
Hi, 
I heard about simon.py on the python bytes podcast.

I have tried to install simon.py under OSX 10.12.3 and python 3.6 as you recommend. There [psutils fails to install and I have reported that as an issue to psutils github repo.](https://github.com/giampaolo/psutil/issues/992)

I can however get the packages to install using python 2.7.10 . However the app does not show any data and appears blank.

![image](https://cloud.githubusercontent.com/assets/11098/24084784/4fc72214-0cc7-11e7-9de8-cdf2a1f45e25.png)

I was wondering how to troubleshoot . 
I installed simon.py in a virtualenv. Sorry to submit this as an issue , but I didnt know how to get it working. 

```
(simon2) Haris-MBP:simon hari$ pip install psutil
Collecting psutil
Installing collected packages: psutil
Successfully installed psutil-5.2.0
(simon2) Haris-MBP:simon hari$ pip install -r requirements.txt
Requirement already satisfied: psutil==5.2.0 in /Users/hari/.virtualenvs/simon2/lib/python2.7/site-packages (from -r requirements.txt (line 1))
Collecting pyobjc-core==3.2.1 (from -r requirements.txt (line 2))
  Using cached pyobjc_core-3.2.1-cp27-cp27m-macosx_10_6_intel.whl
Collecting pyobjc-framework-Cocoa==3.2.1 (from -r requirements.txt (line 3))
  Using cached pyobjc_framework_Cocoa-3.2.1-cp27-cp27m-macosx_10_6_intel.whl
Installing collected packages: pyobjc-core, pyobjc-framework-Cocoa
Successfully installed pyobjc-core-3.2.1 pyobjc-framework-Cocoa-3.2.1
(simon2) Haris-MBP:simon hari$ python simon.py
```

**Comment by harijay** [comment]
Hi,
I fixed it for python 2.7
Had to change the tuple lookup to use an int

I now have Simon working and showing statistics...great application. Thanks for this

Hari

![image](https://cloud.githubusercontent.com/assets/11098/24085133/73064e3a-0ccc-11e7-9799-1a6ce74090b8.png)

```
def bytes2human(n):
    # Credits to /u/cyberspacecowboy on reddit
    # https://www.reddit.com/r/Python/comments/5xukpd/-/dem5k12/
    symbols = (' B', ' KiB', ' MiB', ' GiB', ' TiB', ' PiB', ' EiB', ' ZiB',
               ' YiB')
    i = math.floor((math.log(abs(n)+1, 2)) / 10)
    return '%.1f%s' % (n/2**(i*10), symbols[int(i)])
```

**Comment by half0wl** [comment]
@harijay Thanks! Yep, it seems that `math.floor()` returns a float in Python 2 and we can't use that as a tuple lookup. Would you like to send in a pull request with this fix? I'd be happy to merge it :-)

**Comment by half0wl** [comment]
Closing this, fixed by 9ac5f4d
